content/en/synthetics/private_locations/configuration.md

```diff
@@ -73,6 +73,15 @@ docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetic
 | `concurrency` | Number | `10` | Maximum number of tests executed in parallel. |
 | `maxTimeout` | Number | `60000` | Maximum test execution duration for API tests (in milliseconds). |
 
+## Private Certificates
+
+You can upload custom certificates to your private locations to have your API and Browser tests perform SSL handshake using your own `.pem` files. To do so:
+
+* Mount your certificates (ie. all the relevant `.pem` files) to `/etc/datadog/certs` when spinning up your private location containers. This can be done the same way your private location configuration file is mounted. 
```

Comment:
```suggestion
* Mount your certificates (all the relevant `.pem` files) to `/etc/datadog/certs` when spinning up your private location containers. This can be done the same way your private location configuration file is mounted. 
```